src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,37 @@
+@import "../utils/variables";
+@import "../utils/mixins";
+@import "../utils/functions";
+@import "../normalize/normalize.scss";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $cell-gap;
+  width: 7 * $cell-size + 10 * $cell-gap;
+
+  &__cell {
+    @include square ($cell-size);
+    background-color: #{$cell-color};
+    border: #{$cell-border};
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition: all 0.5s ease;
+    cursor: pointer;
```

Comment:
I'm not sure but I would suggest moving it to :hover on line 28 
if you turn off your hover, pointer will still appear in the cells